app/views/users/index.html.erb

```diff
@@ -6,7 +6,7 @@
     <td><%= sort_link(@q, :email) %></td>
     <td><%= sort_link(@q, :is_male, 'Gender') %></td>
     <td><%= sort_link(@q, :birthday) %></td>
-    <td></td>
+    <td><%= sort_link(@q, :interests) %></td>
```

Comment:
This sorting does not work 😿 But I think that we can skip sorting for that column, it's not required 😄